scripts/exgdas_global_marine_analysis_prep.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+################################################################################
+#  UNIX Script Documentation Block
+#                      .                                             .
+# Script name:         exufsda_global_marine_analysis_prep.py
+# Script description:  Stages files and generates YAML for UFS Global Marine Analysis
+#
+# Author: Guillaume Vernieres      Org: NCEP/EMC     Date: 2022-03-28
+#
+# Abstract: This script stages the marine observations necessary
+#           to produce a UFS Global Marine Analysis.
+#
+# $Id$
+#
+# Attributes:
+#   Language: Python3
+#
+################################################################################
+
+# import os and sys to add ush to path
+import os
+import sys
+import yaml
+import glob
+import dateutil.parser as dparser
+
+# get absolute path of ush/ directory either from env or relative to this file
+my_dir = os.path.dirname(__file__)
+my_home = os.path.dirname(os.path.dirname(my_dir))
+ufsda_home = os.path.join(os.getenv('HOMEgfs'), 'sorc', 'ufs_da.fd', 'UFS-DA')
```

Comment:
A lot of these paths are not valid anymore, but yes, again, on me to fix. This is mainly a reminder that I need to fix develop with feature/cycling ASAP after this is merged.